forte/ontology_specs/race_qa.json

```diff
@@ -3,14 +3,14 @@
   "imports": [
```

Comment:
Are everything in this file included in `base_ontology`? Then let's delete this file.